env/release.mk

```diff
@@ -6,7 +6,7 @@
 export JENKINS_URL=https://cloudbees.ci.cloudbees.com/
 
 # the host to publish bits to
-export PKGSERVER=mirrorbrain@pkg.jenkins.io
+export PKGSERVER=mirrorbrain@52.202.51.185
```

Comment:
why not create an ssh dns record?

Reply:
like pkg.origin.jenkins.io ?

Reply:
This will allow us to request certificate using letsencrypt in the futur

Reply:
yeah